packages/server/test/question.integration.ts

```diff
@@ -198,16 +198,64 @@ describe('Question Integration', () => {
         status: OpenQuestionStatus.Drafting,
       });
 
-      const response = await supertest({ userId: user.id })
+      await supertest({ userId: user.id })
         .post('/questions')
         .send({
           text: 'i need to know where the alamo is',
           queueId: queue.id,
           questionType: QuestionType.Bug,
           force: true,
-        });
+        })
+        .expect(201);
+    });
+    it('lets student (who is TA in other class) create question', async () => {
+      const user = await UserFactory.create();
+
+      // Make user a TA in other class
+      const queueOther = await QueueFactory.create({});
+      await TACourseFactory.create({
+        userId: user.id,
+        courseId: queueOther.courseId,
+      });
+
+      // Make them student
+      const queue = await QueueFactory.create({ allowQuestions: true });
+      await StudentCourseFactory.create({
+        userId: user.id,
+        courseId: queue.courseId,
+      });
+
+      await supertest({ userId: user.id })
+        .post('/questions')
+        .send({
+          text: 'i need to know where the alamo is',
+          queueId: queue.id,
+          questionType: QuestionType.Bug,
+          force: false,
+        })
+        .expect(201);
+    });
+    it('works when other queues and courses exist', async () => {
```

Comment:
Do we need this test if "other queue's and courses" are present in the previous test? It doesn't hurt, just seems like it might already be covered

Reply:
yeah, when they failed, they failed differently (404 vs 401) so I made both tests